tests/responses/dnb/test_splash.html

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
This one is not yet used in the tests. Perhaps you can add a test that checks `scrape_for_abstract` and use `responses` library to fake the request to the site with the content of `test_splash.html`. https://github.com/getsentry/responses#response-body-as-string